blueprints/automation/unifiprotect/push_notification_smart_event.yaml

```diff
@@ -315,6 +333,16 @@ action:
                   [{% if input_silence_timer > 0 %}
                   { "action": "{{ silence_action }}", "title": "Silence" },
                   {% endif %}]
+  - choose:
+      - conditions: "{{ input_send_telegram }}"
+        sequence:
+          - service: notify.{{ input_notify_telegram }}
+            data:
+              title: "{{ notification_title }}"
+              message: "{{ notification_message }}"
+              data:
+                photo:
+                  url: "{{ base_url }}{{ notification_image }}"
```

Comment:
`notification_image` is already `input_base_image_url` + `state_attr(camera_entity_id, 'entity_picture')`

Using base URL is redundant here. Also, `base_url` is your "internal" HA URL, `base_image_url` is your "external" HA URL.

Reply:
Thanks for the feedback!
Good call on this one. I'll remove base_url where it's used. 
My HA setup does not have an external IP (hence using Telegram to punch out), however, it looks like a local URL will still work with input_base_image_url.
Also for Telegram to work it's not optional.